src/symbols.js

```diff
@@ -736,6 +736,16 @@ var symbols = {
         },
 
         // AMS Misc
+        "\\#": {
```

Comment:
Oops, sorry – can you also move these up to the "// Misc symbols" section with the other main-font ones?